dbt/adapters/snowflake/connections.py

```
@@ -18,10 +18,10 @@
     InternalException, RuntimeException, FailedToConnectException,
     DatabaseException, warn_or_error
 )
-from dbt.adapters.base import Credentials
+from dbt.adapters.base import Credentials  # type: ignore
```

attr-defined errors are masked by this `# type: ignore` declaration